eng/versioning/pom_file_version_scanner.ps1

```diff
@@ -57,6 +57,10 @@ $DependencyTypeForError = "$($DependencyTypeCurrent)|$($DependencyTypeDependency
 $UpdateTagFormat = "{x-version-update;<groupId>:<artifactId>;$($DependencyTypeForError)}"
 $UseVerboseLogging = $PSBoundParameters['Debug'] -or $PSBoundParameters['Verbose']
 
+if ($UseVerboseLogging) {
```

Comment:
Does this get set when we check `use diagnostic logging` in the pipeline and/or would that be a useful thing to support for this case? (if so should be a one line change I can show)